components/husky_a300/husky_a300_lockout_tagout.mdx

```diff
@@ -37,3 +39,20 @@ We cannot completely discharge these batteries without damaging them, but we can
     :::
 
 </details>
+
+<Admonition type="danger" title="Safe Maintenance Practices">
+  Follow the guidance below while performing maintenance to ensure that the system is
+  in a safe state following any maintenance work on the robot.
+
+  1. Clean up all tools and ensure that no tools or installation materials have been left inside the robot.
+  1. Ensure that all cables have been tightened to the specified torque values and that there are no loose or unconnected cables.
+  1. Reinstall any fuse covers that were removed.
+  1. Restore all terminal boots to their original position prior to maintenance.
+
+  Once all parts have been reinstalled and maintenance is completed, perform the following system checks:
+  1. Power on the robot.
+  1. Connect the manual charger and charge the robot for at least 5 minutes. Confirm that the state of charge is increasing
+     and that there are no unexpected sounds or smells.
+  1. With the robot in an open area at least 10 m in diameter and away from other people, connect the joystick and drive the
+     robot for at least 30 seconds to confirm it is performing as expected.
```

Comment:
Consider adding a check for E-stop functionality, including the wireless transmitter and rear charge port door.